examples/plot_simulate_evoked.py

```diff
@@ -63,6 +67,16 @@
 
 params.update({'sync_evinput': True})
 net_sync = Network(params)
-dpls_sync = simulate_dipole(net_sync, n_jobs=1, n_trials=1)
+
+###############################################################################
+# Next, let's simulate a single trial using the MPI backend. This will
+# start the simulation trial across the number of processors (cores)
+# specified by n_procs using MPI. The 'mpiexec' launcher is for
+# openmpi, which must be installed on the system
+from hnn_core import MPIBackend
```

Comment:
I think it's better to do:

```py
from hnn_core.parallel import MPIBackend
```

but I'm fine pushing this to another PR

Reply:
Then the user has to know about `.parallel` and structuring of the files.

Reply:
no strong feeling. But we shouldn't put everything in global namespace. We can document some of the stuff in API and through examples, so it's just copy-paste for users.

Reply:
Okay, I've just been following your lead on what to name things.